arkouda/random/__init__.py

```diff
@@ -1,9 +1,8 @@
-from ._generator import Generator, default_rng
+from ._generator import Generator, default_rng  # noqa
```

Comment:
I don't understand the comment # noqa

Reply:
it tells flake8 to ignore this line because it complains about `default_rng` being imported but not included in the `__all__`

I'm using it here because we want `default_rng` to be visible to the user only under the `Random` module. So you can call `ak.Random.default_rng()`, but not `ak.default_rng()`. This was the easiest way I saw to get this behavior, but I'm open to suggestions!

Reply:
I googled it and it looks like there is another option:
https://stackoverflow.com/questions/31079047/python-pep8-class-in-init-imported-but-not-used

Change setup.cfg
to add `__init__.py: F401`
like this:

```
[flake8]
max-line-length = 105
extend-ignore = E203
per-file-ignores =
    PROTO_tests/tests/operator_test.py: E501
    PROTO_tests/tests/symbol_table_test.py: F841
    __init__.py: F401

```

But either way is fine with me.

Reply:
i think leaving it as the `# noqa` is the way to go, since we could want that error on other lines in the file if we accidentally left them out of the `__all__`. It's just this one specifically that is the exception to the rule

Reply:
Sounds good.